build.cake

```diff
@@ -49,7 +49,7 @@ var NetCoreTests = new String[]
 //////////////////////////////////////////////////////////////////////
 
 var PROJECT_DIR = Context.Environment.WorkingDirectory.FullPath + "/";
-var PACKAGE_DIR = PROJECT_DIR + "package/";
+var PACKAGE_DIR = Argument("artifact-dir", PROJECT_DIR + "package") + "/";
```

Comment:
Allows overriding the `package` path from the command line, which you can see here:
https://github.com/nunit/nunit/pull/3096/files#diff-fec826feae04e51c0d94076385408bdcR24